docs/guide/install.rst

```diff
@@ -246,23 +247,7 @@ following:
    prepend-path PKG_CONFIG_PATH $install_root/share/pkgconfig
 
 
-3. Install into a Specific Path
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-You may eventually install Verilator into a specific installation prefix,
-as most GNU tools support:
```

Comment:
I think this does work, and is what distro authors need to use, can you comment on removal - was it confusing?

Reply:
It does work but it's a duplicate of the previous section.